cypress-tests/cypress/support/commands/projects.ts

```diff
@@ -69,39 +69,46 @@ interface NewProjectProps extends ProjectIdentifier {
 }
 
 function createProject(newProjectProps: NewProjectProps) {
```

Comment:
This is now a misnomer. Use something like `getOrCreateProject()`

Reply:
Sure I will rename it.

Reply:
I went for `createProjectIfMissing`.